src/com/sun/ts/tests/ejb32/mdb/modernconnector/connector/EventMonitorAdapter.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2013, 2018 Oracle and/or its affiliates. All rights reserved.
```

Comment:
This file doesn't need the copyright date change any more.

Reply:
Fixed